index.js

```diff
@@ -7,7 +7,8 @@
   var xmlParser = require('xml2json');
   var FundamentalAccountingConcepts = require('./FundamentalAccountingConcepts.js');
 
-  function parse(pathToXbrlDoc) {
+  function parse(pathToXbrlDoc, output) {
+    output = (output=== null)? false : output;
     var self = this;
```

Comment:
The reason why I set it as such is that if output === true, output should remains true. False is a "default value". Using ternary operator wouldn't work in that sense.